x/profiles/handle_msg.go

```diff
@@ -22,39 +18,12 @@ func (m *Module) HandleMsgExec(index int, _ *authz.MsgExec, _ int, executedMsg s
 }
 
 // HandleMsg implements modules.MessageModule
-func (m *Module) HandleMsg(index int, msg sdk.Msg, tx *juno.Tx) error {
+func (m *Module) HandleMsg(_ int, msg sdk.Msg, tx *juno.Tx) error {
```

Comment:
The IBC messages should be able to replace to event handler since these operation also emit events.
See:
https://github.com/desmos-labs/desmos/blob/master/x/profiles/ibc_module.go#L242
https://github.com/desmos-labs/desmos/blob/master/x/profiles/ibc_module.go#L273
https://github.com/desmos-labs/desmos/blob/master/x/profiles/ibc_module.go#L316
https://github.com/desmos-labs/desmos/blob/master/x/profiles/ibc_module.go#L365